src/project/modals/signup.modal.js

```diff
@@ -0,0 +1,99 @@
+        await this.elements.lastNameField().fill('123');
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toHaveText('Last name is invalid');
+        await this.elements.lastNameField().clear();
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toHaveText('Last name required');
+        await this.elements.lastNameField().fill('k');
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toHaveText('Last name has to be from 2 to 20 characters long');
+        await this.elements.lastNameField().clear();
+        await this.elements.lastNameField().fill(registrationData.last_name);
+    }
+
+    async checkEmailField(registrationData){
+        await this.elements.emailField().fill('123');
+        await this.elements.lastNameField().click();
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toHaveText('Email is incorrect');
+        await this.elements.emailField().clear();
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toHaveText('Email required');
+        await this.elements.emailField().fill(registrationData.email);
+    }
+
+    async checkPasswordFields(registrationData){
+        await this.elements.passwordField().fill('145Ktest');
+        await this.elements.repeatPasswordField().fill(registrationData.password);
+        await this.elements.passwordField().click();
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toHaveText('Passwords do not match');
+        await this.elements.passwordField().clear();
+        await this.elements.passwordField().fill(registrationData.password);
+        await this.elements.passwordField().clear();
+        await this.elements.passwordField().fill('2332');
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toContainText('Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter');
+        await this.elements.passwordField().clear();
+        await expect(this.elements.invalidFeedback()).toHaveCSS('color', 'rgb(220, 53, 69)');
+        await expect(this.elements.invalidFeedback()).toHaveText('Password required');
+        await this.elements.passwordField().clear();
+        await this.elements.passwordField().fill(registrationData.password);
+        await this.elements.repeatPasswordField().fill(registrationData.password);
+    }
```

Comment:
each of these methods does care a lot, each method here has not only one responsibility

I can't figure out what is going on inside just reading method name